euphony/src/main/java/co/euphony/common/Constants.java

```diff
@@ -20,6 +22,7 @@ public class Constants {
 	public static final int MIN_REF = 50;
 
 	public static final int DEFAULT_REF = 500;	// BASE Reference value
+	public static final String REQUIRED_PERMISSION = Manifest.permission.RECORD_AUDIO;
```

Comment:
I think this variable is not used :open_mouth: If it is not used, it would be good to remove this line :)